jupyterhub/templates/hub/configmap.yaml

```diff
@@ -255,6 +255,9 @@ data:
   {{- if .Values.scheduling.userScheduler.enabled }}
   singleuser.scheduler-name: "{{ .Release.Name }}-user-scheduler"
   {{- end }}
+  {{- if .Values.scheduling.podPriority }}
```

Comment:
.podPriority.enabled?

Reply:
Woo this have been truthy during testing for sure already. I think it triggered a silent failure on the kubernetes API server that simply stripped it away when no such priorityClass was found. Normally it sets the `priority` field after reading the priorityClassName, and you cannot modify that in retrospect without recreating the pod.